wiki/ProjectBestPractices.md

```diff
@@ -0,0 +1,128 @@
+
+## Code Styles
+
+- Use `get_node` instead of `$` to get a node from the tree, for example, `@onready var my_node = get_node("MyNode")`, (based: https://github.com/FlamingoFiestaStudio/OlympusGoneWild/pull/26#discussion_r1328143299);
+- Variables names should be in snake_case, example: `var my_variable_name: int = 42`;
+- Try to implicit all types, for example: `var my_string: string = "My String"`;
+
+## Branch Naming Convention
+
+The branch should have the number of the issue and the title of the issue. For example: `123-my-issue-title`.
+
+To simplify, you could just click here on the issue page, and this will do the magic for you:
+
+![Create Branch](./images/Create-Branch.png)
+
+## Commit Message Convections
+
+Consistently using this convention ensures clear and structured commit messages, making it easier for team members to understand the purpose of each commit.
+
+`<type>[optional scope]: <description>`
+
+- `<type>`: Describes the purpose or category of the commit;
+- `[optional scope]` (optional): Specifies the scope or affected component of the commit or issue number;
+- `<description>`: Provides a brief, concise summary of the changes introduced in the commit;
+
+### Commit Types
+
+Choose an appropriate <type> from the following list:
+
+- `feat`: A new feature or enhancement;
+- `fix`: A bug fix or correction;
+- `docs`: Documentation changes or updates;
+- `style`: Code style and formatting improvements (no functional changes);
+- `refactor`: Code refactoring, restructuring, or optimizations;
+- `chore`: Routine tasks, maintenance, or tooling changes;
+
+### Commit Description
+
+The <description> should be a concise summary of the changes introduced in the commit. Be clear and informative but keep it brief.
+
+### Example Commit Messages
+
+Here are some examples of correctly formatted commit messages:
+
+- `feat(#123): Add a new player bullet.`
+- `fix(player): Fix player movimentation issue.`
+- `docs(readme): Update installation instructions.`
+- `style(main menu): Adjust formatting for consistency.`
+- `refactor: Optimize enemy code.`
+- `chore: Update dependencies to latest versions.`
+
+## Pull Requests
+
+On creating a pull request, we have a [basic pull request template with some sections](https://github.com/FlamingoFiestaStudio/OlympusGoneWild/blob/main/.github/PULL_REQUEST_TEMPLATE.md), try to fill all of them.
+
+If you don't have nothing to add in the description of some section, please leave as: `N/A.` (Not Applicable).
+
+### Comments Resolution and revisions
+
+When you submit a PR, someone will review your code:
+
+- If that person approves your changes, you did a nice work;
+- If that person leaves some comments:
+    - If you agree, change what the reviewers said, and notify the person after push;
+    - If you do not agree, be nice and explain why do you not think in the same way;
+    - The comments should be only closed by the person who left the comment;
```

Comment:
I think you could add that it doesn't necessarily needs to be the person who commented it if its something simple since this can lead to unnecessary waiting

Reply:
I was thinking about that, I should have added another point where if the person doesn't say anything in a week the owner of the PR can close it.

What you think?

Reply:
yes sounds good